Ginger/Ginger/Reports/ValidationRules/ValidateEmptyValueWithDependency.cs

```diff
@@ -0,0 +1,103 @@
+#region License
+/*
+Copyright © 2014-2022 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using amdocs.ginger.GingerCoreNET;
+using GingerCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+using System.Windows.Data;
+
+namespace Ginger.ValidationRules
+{
+    public class ValidateEmptyValueWithDependency : ValidationRule
+    {
+        private string _Message = string.Empty;
+        private object _ConfigurationObj = null;
+        private string _DependentObjField = string.Empty;
+
+        public ValidateEmptyValueWithDependency(string message = "Value must be provided")
```

Comment:
this constructor is not valid for this validation, dependencyObj and dependecyField must be provided